python_modules/libraries/dagster-dbt/dagster_dbt/rpc/resources.py

```diff
@@ -316,28 +317,26 @@ def seed(
         if kwargs is not None:
             data["params"]["task_tags"] = kwargs
 
-        return self._post(data=json.dumps(data))
+        return self._get_result(data=json.dumps(data))
 
     def generate_docs(
         self,
-        models: List[str] = None,
-        exclude: List[str] = None,
+        compile: bool = False,
```

Comment:
The previous params (models and exclude) are not valid parameters to the true RPC dbt call, causing errors when these parameters are provided. My fix corrects the parameters to match the [dbt docs](https://docs.getdbt.com/reference/commands/rpc/#generate-docs).

Reply:
good catch! do you mind removing these parameters from the dbt_resource.py definition (base class) and the cli resource? I checked and these are not valid in the cli version of the command either, so this is just a straight up bug.

Reply:
done!